src/main/java/bobby/command/ToDoCommand.java

```diff
@@ -0,0 +1,62 @@
+package bobby.command;
+
+import bobby.Storage;
+import bobby.Ui;
+import bobby.exception.BobbyException;
+import bobby.exception.ToDoException;
+import bobby.task.TaskList;
+import bobby.task.ToDo;
+
+
+/**
+ * Represents a 'todo' command
+ */
+public class ToDoCommand extends Command {
+    /** The full user input command */
+    private String fullCommand;
+    /** The full user input command in array form */
+    private String[] fullCommandArr;
```

Comment:
I think it might be better to name it in a plural form such as fullUserCommands 😄